pkg/netutil/networkwalker.go

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Should this be in `pkg/idutil/netwalker` for consistency with other objects?
https://github.com/containerd/nerdctl/tree/main/pkg/idutil

Reply:
Hi @AkihiroSuda, I was thinking about this too, but `networkConfig` is not exported outside of `netutil` package, so the only solution I could figure out was to implement the walker in the same package as the struct.

Reply:
@AkihiroSuda @Zheaoli Addressed, please review

Reply:
@Mihai22125  @Zheaoli   No LGTM for  public `networkConfig`. PTAL https://github.com/containerd/nerdctl/pull/1538#discussion_r1030761472 !